src/Workspaces/VisualBasic/Portable/LanguageServices/VisualBasicSyntaxFactsService.vb

```diff
@@ -662,7 +662,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     Return Nothing
                 End If
 
-                Return TextSpan.FromBounds(method.BlockStatement.Span.End, method.EndBlockStatement.SpanStart)
```

Comment:
Not necessarily, I just wanted to make sure I understood.

I think your comment explains it well and covers your intent, so I'd definitely keep that regardless. With the intent then clear, either implementation is acceptable, and it's probably just a matter of preference (or, if you're sufficiently paranoid, these do have different default behaviors if some new type of syntax was allowed between the BlockStatement and first contained Statement in some future language version).